xclim/data/variables.yml

```diff
@@ -113,6 +113,11 @@ variables:
     standard_name: sea_ice_area_fraction
     data_flags:
       - percentage_values_outside_of_bounds:
+  smd:
+    canonical_units: mm
```

Comment:
In the function, you put "[precipitation]" as the dimensions of `smd` and then you convert it to `mm/d`. Shouldn't then that be the canonical units?

Reply:
Would it be appropriate to add something like:
```
units.define("[soil_moisture_deficit] = [length] / [time]")
```
to `xclim/core/units.py` and then use that to specify the dimensions? Or is it fine to use "[precipitation]" here given that `smd` and `pr` should always have the same units?